spec/controllers/reservations_controller_spec.rb

```diff
@@ -571,11 +571,19 @@
                                            start_date: Time.zone.today,
                                            due_date: Time.zone.today + 4.days),
               equipment_item: ''
+          # rubocop:disable all
+          time_int = Time.zone.now.to_time.to_i
+          # rubocop:enable all
+          @rounded = Time.zone.at((time_int / 900.0).round * 900)
```

Comment:
I decided to round off to the nearest 15 minutes because: when running the test and comparing two time.now statements that are generated at different times will result in an error.